src/Mod/Path/PathScripts/PathDrilling.py

```diff
@@ -187,8 +180,15 @@ def circularHoleExecute(self, obj, holes):
             self.commandlist.append(Path.Command('G0', {lastAxis: 0.0, 'F': self.axialRapid}))
 
     def opSetDefaultValues(self, obj, job):
-        '''opSetDefaultValues(obj, job) ... set default value for RetractHeight'''
-        obj.RetractHeight = 10.0
+        '''opSetDefaultValues(obj, job) ... set default value for PeckRetractHeight'''
+        
+        PeckDepthDefaultExpression   = 'OpToolDiameter*0.75'
+        PeckRetractDefaultExpression   = 'OpStartDepth+1mm'
+        
+        if self.applyExpression(obj, 'PeckRetractHeight', PeckRetractDefaultExpression):
+                obj.PeckRetractHeight = 10
+        if self.applyExpression(obj, 'PeckDepth', PeckDepthDefaultExpression):
+                obj.PeckRetractHeight = 1
```

Comment:
This breaks the SetupSheet processing of the two values. You need to set the value from the setup sheet if it exists. See `EnableRotation` a little farther down for an example.

Reply:
Done, also for Dwell time.